docker-compose.yml

```diff
@@ -1,21 +1,33 @@
 version: '3'
+
 services:
-  ms:
+
+  metasploit:
+    container_name: metasploit
```

Comment:
Hey, just a quick question on this line - is it possible to remove this `container_name` attribute and use `docker-compose exec` instead of `docker exec` to resolve containers?

```
docker-compose exec metasploit docker/entrypoint.sh ./msfconsole
```

Note that this would work in a similar way to the `docker exec` command that you've added a demo for.
Would that still work with your use case? 🤔

Reply:
@adamgalway-r7  Yes we can remove the `container name` attribute but the way of execute is diferent

Without `container_name`

![dockerpserror](https://user-images.githubusercontent.com/28166400/85903928-2713a900-b7cd-11ea-88f9-55489c5e48b9.png)

With `container_name`:

![withcontainername](https://user-images.githubusercontent.com/28166400/85903990-56c2b100-b7cd-11ea-821d-cfa29714f783.png)

if we remove the `container_name` we have to access with the name or id of the container